test/unit/collection-view.empty-view.spec.js

```diff
@@ -279,7 +279,7 @@ describe('collectionview - emptyView', function() {
       });
 
       it('renders other empty view instance', function() {
-        expect(this.getEmptyViewStub).to.have.been.calledWith(this.collectionView.children.first());
+        expect(this.getEmptyViewStub).to.have.been.called;
```

Comment:
So this happened after some dependency updates.  I am not 100%, but I expect (pun intended)  that this is a later version of chai failing the test.   `getEmptyView` is not called with _any_ arguments.